design/proposals/load_policy/load_policy_validation.md

```diff
@@ -0,0 +1,148 @@
+# Solution Design - Load policy validation
+
+## Table of Contents
+TBD
+
+## Glossary
+TBD
+
+## Useful Links
+- [Feature doc](https://ca-il-jira.il.cyber-ark.com:8443/browse/ONYX-8566)
+- [UX research page](https://ca-il-confluence.il.cyber-ark.com.mcas.ms/display/rndp/JWT+Authenticator+-+load+policy+validation)
+- [JWT authenticator stage 1.1 feature doc](https://ca-il-confluence.il.cyber-ark.com.mcas.ms/display/rndp/JWT+Authenticator+-+re-scope#JWTAuthenticatorrescope-Stage1.1-SimplegenericJWTauthenticatorwithtokenschema)
+
+
+## Issue Description
+In misconfiguration errors, for example:
+hosts with empty JWT annotation value, we are failing at JWT authentication request which is bad UX.
```

Comment:
i would emphasize that we are returning a 401 response in such a case which is a client error but the error is not in the request itself rather than in the policy load. I would also add that we accept loading a policy that is worthless and failing it would help the customer to create valid policies

Reply:
thanks